packages/main/src/RatingIndicator.js

```diff
@@ -233,9 +249,13 @@ class RatingIndicator extends UI5Element {
 		}
 
 		const down = isDown(event) || isLeft(event);
-		const up = isRight(event) || isUp(event) || isSpace(event) || isEnter(event);
+		const up = isRight(event) || isUp(event);
```

Comment:
It'd be great if those values have semantic meaning.
This means that instead of the obvious to name them after their puprpose. For example, **_up_** could become **_isIncrease_**

Reply:
Done